tests/framework/test_state_machine.py

```diff
@@ -40,14 +39,22 @@ def inner(self, pop_data):
     return pop_fixture()
 
 
+def get_context(components, configuration=None):
+    simulation = SimulationContext(components=components,
```

Comment:
To replace with the interactive context once it's smart.

Reply:
and you're leaving this same function in multiple places because it's shortly going to be replaced right?

Reply:
Yep